pytest_timeout.py

```diff
@@ -433,6 +432,43 @@ def dump_stacks():
         write("".join(traceback.format_stack(frame)))
 
 
+def _getdimensions():
+
+    if sys.version_info >= (3, 3, 0):
+        import shutil
```

Comment:
Could we move the imports outside of the function together with the other imports?  You can still use the if condition on the imports.  Having the import machinery invoked in functions is not nice, it makes it much harder to know when what is happening and can result in weird locking issues and hard to debug circular import issues etc.